src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogOciConnectionFormatForm.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs\Communication\Form;
+
+use Generated\Shared\Transfer\PunchoutCatalogConnectionTransfer;
+use Spryker\Zed\Kernel\Communication\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+/**
+ * @method \SprykerEco\Zed\PunchoutCatalogs\PunchoutCatalogsConfig getConfig()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Persistence\PunchoutCatalogsRepositoryInterface getRepository()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Business\PunchoutCatalogsFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\PunchoutCatalogs\Communication\PunchoutCatalogsCommunicationFactory getFactory()
+ */
+class PunchoutCatalogOciConnectionFormatForm extends AbstractType
+{
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     * @param array $options
+     *
+     * @return void
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $this->addUsernameField($builder)
+            ->addPasswordField($builder);
+    }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return $this
+     */
+    protected function addUsernameField(FormBuilderInterface $builder)
+    {
+        $builder->add(PunchoutCatalogConnectionTransfer::USERNAME, TextType::class, [
```

Comment:
This approach looks interesting, but why you've chosen a different way, other than in `PunchoutCatalogCXmlConnectionFormatForm`? There you used additional constants for names, here - you use the transfer field names.

As for me, it creates some kind of inconsistency. Please stick to 1 way or to another.
If this is a similar functionality, why your implementation of common things differs?

Reply:
Both `PunchoutCatalogCXmlConnectionFormatForm` and `PunchoutCatalogOciConnectionFormatForm` uses `PunchoutCatalogConnectionTransfer` properties as field names, please double check